app/docs/dev/policies/meshtrafficpermission.md

```diff
@@ -0,0 +1,290 @@
+---
+title: MeshTrafficPermission (beta)
+---
+
+{% warning %}
+This policy uses new policy matching algorithm and is in beta state,
+it should not be mixed with [TrafficPermission](../traffic-permissions.md).
+{% endwarning %}
+
+## TargetRef support matrix
+
+| TargetRef type    | top level | to  | from |
+| ----------------- | --------- | --- | ---- |
+| Mesh              | ✅        | ✅  | ❌   |
+| MeshSubset        | ✅        | ✅  | ❌   |
+| MeshService       | ✅        | ✅  | ❌   |
+| MeshServiceSubset | ✅        | ✅  | ❌   |
+| MeshGatewayRoute  | ✅        | ❌  | ❌   |
+
+If you don't understand this table you should read [matching docs](/docs/{{ page.version }}/policies/matching).
+
+## Configuration
+
+### Action
+
+{{ site.title }} allows configuring one of 3 actions for a group of service's clients:
```

Comment:
Should this be `site.mesh_product_name` (maybe just add it in `jekyll.yaml` like in #1122)